modules/multiplayer/multiplayer_spawner.cpp

```diff
@@ -86,6 +86,23 @@ void MultiplayerSpawner::_get_property_list(List<PropertyInfo> *p_list) const {
 	}
 }
 #endif
+
+TypedArray<String> MultiplayerSpawner::get_configuration_warnings() const {
+	TypedArray<String> warnings = Node::get_configuration_warnings();
+
+	if (spawn_path.is_empty() || !has_node(spawn_path)) {
+		warnings.push_back(RTR("A valid NodePath must be set in the \"Spawn Path\" property in order for MultiplayerSpawner to be able to spawn Nodes."));
+	}
+	bool has_scenes = get_spawnable_scene_count() > 0;
+	// Can't check if method is overriden in placeholder scripts.
+	bool has_placeholder_script = get_script_instance() && get_script_instance()->is_placeholder();
+	if (!has_scenes && !GDVIRTUAL_IS_OVERRIDDEN(_spawn_custom) && !has_placeholder_script) {
```

Comment:
When does `GDVIRTUAL_IS_OVERRIDDEN()` return true?
Any empty script removes the warning.

Reply:
`GDVIRTUAL_IS_OVERRIDDEN` returns true the class has an extension or script which implements that method.

But as per the comment:
```
// Can't check if method is overridden in placeholder scripts.
```

So if the script is built-in, or can't be checked by the editor (not yet saved, can't be parsed, etc), a placeholder script will be used, thus forcing the check to pass.